doc/Makefile.am

```diff
@@ -285,18 +285,33 @@ sphinx_verbose = $(sphinx_verbose_$(V))
 sphinx_verbose_ = $(sphinx_verbose_$(AM_DEFAULT_VERBOSITY))
 sphinx_verbose_0 = @echo "  BUILD     manpages";
 
+sphinx_verbose_flags = $(sphinx_verbose_flags_$(V))
+sphinx_verbose_flags_ = $(sphinx_verbose_flags_$(AM_DEFAULT_VERBOSITY))
+sphinx_verbose_flags_0 =
+sphinx_verbose_flags_1 = -v
+sphinx_verbose_flags_2 = -vv
+
 STDERR_DEVNULL = $(stderr_devnull_$(V))
 stderr_devnull_ =  $(stderr_devnull_$(AM_DEFAULT_VERBOSITY))
 stderr_devnull_0 = >/dev/null 2>&1
 
 $(MAN_FILES): conf.py $(RST_FILES)
-	$(sphinx_verbose)$(PYTHON) -m sphinx -M man $(srcdir) ./ $(STDERR_DEVNULL)
+	$(sphinx_verbose) \
+	SPHINX_BUILDDIR=$(abs_builddir) $(PYTHON) \
+		-m sphinx $(sphinx_verbose_flags) -b man $(srcdir) ./man \
+		$(STDERR_DEVNULL)
 	@echo "  MV        manpages"; \
 	for sec in 1 3 5 7; do \
 	  $(MKDIR_P) man$$sec && \
 	  mv -f $(abs_builddir)/man/*.$$sec man$$sec/; \
 	done
 
+.PHONY: html
+html: conf.py $(RST_FILES)
+	SPHINX_BUILDDIR=$(abs_builddir) $(PYTHON) \
```

Comment:
Prefix with `$(sphinx_verbose)`?

Reply:
That variable is set to echo `BUILD manpages`, which is a bit of a misnomer for the html.  Should I create a new variable that echos `BUILD html`?

Reply:
Ah, good point. You can do that or leave as is - up to you. It isn't too important...

Reply:
Sounds good.  Just did a squash of my old fixup commit and amended the commit at the top of the commit stack (which added the `html` target) to include a new variable that echos `BUILD html`.